Xamarin.PropertyEditing/Common/FilePath.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Xamarin.PropertyEditing
```

Comment:
This should have the `.Common` namespace.

Reply:
done